builder/image/templates/builder

```diff
@@ -108,8 +102,13 @@ if [ ! -f Dockerfile ]; then
     BUILD_OPTS+=' -v '
     BUILD_OPTS+=$(echo $CACHE_DIR)
     BUILD_OPTS+=':/tmp/cache:rw'
-    # give non-root slugbuilder user R/W perms for docker volumes
-    chown -R 2000:2000 $TMP_DIR $CACHE_DIR
+    # give slug group ownership of TMP_DIR and CACHE_DIR.
+    chown -R :2000 $TMP_DIR
+    chown :2000 $CACHE_DIR
```

Comment:
Just thought I'd comment on this to avoid any potentional questions on why I had to do it this way:

`chown -R :2000` failed on `CACHE_DIR` since files in that folder is owned by `slug:slug` with permissions `rwxr-xr-x`. As such running recursive chown in this folder resulted in permission denied.

I doubt it will be any issue though, this folder seems only used by slugbuilder.